backend/src/routes/api/notebooks/notebookUtils.ts

```diff
@@ -52,6 +52,16 @@ export const getNotebook = async (
   }
 };
 
+const checkPodContainersReady = (pod: V1Pod): boolean => {
```

Comment:
This file's changes is related to #520 -- when I started fetching the Notebook faster, it definitely lead me into a situation where the app wouldn't start more frequently. So I adopted this chunk of code to see if everything would gel together -- and it does.